Model/Checkout/SetAmwalOrderDetails.php

```diff
@@ -47,6 +47,7 @@ public function execute(OrderInterface $order, string $amwalOrderId, string $tri
         $orderDetails['order_id'] = $order->getIncrementId();
         $orderDetails['order_entity_id'] = $order->getEntityId();
         $orderDetails['order_created_at'] = $order->getCreatedAt();
+        $orderDetails['order_content'] = json_encode($this->getOrderContent($order));
```

Comment:
If you want to encode to json it's a good idea to introduce the `\Magento\Framework\Serialize\Serializer\Json` class to handle the serialization